src/aics-image-viewer/components/shared/ViewerIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Icon } from "antd";
+
+import ICONS from "../../assets/icons";
+
+// antd icons have a habit of appearing off center
+// TODO: come back and verify if this is still an issue after antd upgrade
+const STYLE = { fontSize: "19px", transform: "translateY(-0.5px) translateX(0.5px)" };
+
+/** Wrapper component for easy inclusion of our own custom icons. */
+const ViewerIcon: React.FC<{ type: keyof typeof ICONS }> = ({ type }) => <Icon component={ICONS[type]} style={STYLE} />;
```

Comment:
cool

Reply:
just as devil's advocate, because `ViewerIcon` is so similar to `Icon`, you could just directly import the individual svgs in the TS files that need them and just use `Icon` right?  and you wouldn't need the assets/icons/index.ts file at all.   The icon style could be a shared constant or a css class.

Reply:
Yeah I'm honestly 50/50 on it - this component used to be slightly bigger before I added svgr, and if I'd made it as easy as svgr does in the first place I might not have thought to create it. The things it's giving us though is: 1. specific styling (using that STYLE constant, rather than trying to catch custom icons in a global style that might be leaky; 2. easy addition (add to `icons/index.ts` and that key automatically becomes a valid value for `type` and doesn't have to be imported individually at the site it's used).